src/app/[locale]/list/_components/AasListHeader.tsx

```diff
@@ -5,8 +5,11 @@ export default async function AasListHeader() {
     const t = await getTranslations('aas-list');
 
     return (
-        <Typography variant="h2" textAlign="left" marginBottom={2}>
-            {t('header')}
-        </Typography>
+        <>
+            <Typography variant="h2" textAlign="left" marginBottom={2}>
+                {t('header')}dsafdsffdas
```

Comment:
dsafdsffdas 😆

Reply:
ups, will remove this in the test branch 😄